silx/gui/widgets/test/__init__.py

```diff
@@ -1,7 +1,7 @@
 # coding: utf-8
 # /*##########################################################################
 #
-# Copyright (c) 2016-2018 European Synchrotron Radiation Facility
+# Copyright (c) 2020 European Synchrotron Radiation Facility
```

Comment:
I think for copyright it is best to keep start-end, i.e., 2016-2020 here, but I will merge it anyway.